cmd/policy_controller/main.go

```diff
@@ -63,28 +47,25 @@ var (
 
 func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme.Scheme))
-
+	utilruntime.Must(gatewayapi.AddToScheme(scheme.Scheme))
 	utilruntime.Must(v1alpha1.AddToScheme(scheme.Scheme))
 	utilruntime.Must(certmanv1.AddToScheme(scheme.Scheme))
-	utilruntime.Must(gatewayapi.AddToScheme(scheme.Scheme))
-	utilruntime.Must(clusterv1beta2.AddToScheme(scheme.Scheme))
-	utilruntime.Must(workv1.AddToScheme(scheme.Scheme))
-	utilruntime.Must(clusterv1.AddToScheme(scheme.Scheme))
-
-	//+kubebuilder:scaffold:scheme
 }
 
 func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var probeAddr string
-	var certProvider string
+	var ocmHub bool
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
-	flag.StringVar(&certProvider, "cert-provider", "glbc-ca", "The name of the certificate provider to use")
+
+	flag.BoolVar(&ocmHub, "ocm-hub", true,
+		"tells the controller if it is in an ocm hub"+
+			"setting this to false will cause the controller to stop watching for managedcluster resources")
```

Comment:
This seems fine as a temp workaround but once https://github.com/Kuadrant/multicluster-gateway-controller/issues/622 is done I'd expect this to be a more generic option that would allow any gvk to be specified as the "Cluster" resource to watch.